conjure-java-undertow-runtime/src/test/java/com/palantir/conjure/java/undertow/runtime/ConjureExceptionHandlerTest.java

```diff
@@ -74,19 +69,20 @@ public void after() {
     @Test
     public void handlesServiceException() throws IOException {
         exception = new ServiceException(ErrorType.CONFLICT, SafeArg.of("foo", "bar"));
-        Response response = execute();
-        assertThat(response.body().string())
+        HttpURLConnection connection = execute();
+
+        assertThat(connection.getResponseCode()).isEqualTo(ErrorType.CONFLICT.httpErrorCode());
```

Comment:
For all of these tests I needed to get the response code to cause it to "execute" the request otherwise reading the stream was null. In terms of the testing, the ordering seemed immaterial.